src/run_task.py

```diff
@@ -233,7 +232,7 @@ def main(
             ).run()
         except Exception as e:
             log.exception(f"Failed to process with error: {e}")

```

Comment:
Just check this, I think logging an exception re-raises it.

Maybe do a `log.warning`.